flink-java/src/test/java/org/apache/flink/api/java/sca/UdfAnalyzerTest.java

```diff
@@ -62,6 +62,10 @@
 @SuppressWarnings("serial")
 public class UdfAnalyzerTest {
 
+	private static TypeInformation<Tuple2<String, Integer>> stringIntTuple2TypeInfo = TypeInformation.of(new TypeHint<Tuple2<String, Integer>>(){});
```

Comment:
These fields are constants, so they should be final. Can you add the modified and rename the fields to match the naming convention?